docs/Kangal-user-flow.md

```diff
@@ -22,6 +22,7 @@ curl -X POST http://${KANGAL_PROXY_ADDRESS}/load-test \
   -F testData=@./artifacts/loadtests/testData.csv \
   -F envVars=@./artifacts/loadtests/envVars.csv \
   -F type=JMeter
+  -F overwrite=true
```

Comment:
Please add some description of what does this key mean

Reply:
Where should I write this description ? There is no description for other files too ? Should I create a table like accepted values etc?

Reply:
Would be nice to put some description under https://github.com/hellofresh/kangal/blob/master/docs/Kangal-user-flow.md#create section, what do you think?

Reply:
Put a table : https://github.com/hellofresh/kangal/blob/dbcf378e0c70e92f427f98ed2913f4fc8b645314/docs/Kangal-user-flow.md